test/bot.coffee

```diff
@@ -254,3 +255,68 @@ describe 'Robot.react', ->
     @slackbot.robot.react matcher, @handleReaction
     listener = @slackbot.robot.listeners.shift()
     listener.matcher(@reactionMessage).should.be.false
+
+describe 'Users data', ->
+  it 'Should add a user data', ->
+    @slackbot.userChange(@stubs.user)
+
+    user = @slackbot.robot.brain.data.users[@stubs.user.id]
```

Comment:
instead of accesing the brain data directly, wouldn't it make more sense to use the `brain.userForId()` method?